### PR TITLE
Admin dashboard routes

### PR DESCRIPTION
**Pull request by miketillman95**
# Description

Fixes #

- What work was done?
-Created a Stories routes and model for an admin dashboard that will be created in the future, currently contains the basic crud commands
- Why was this work done?
- This work was assigned to start the change the process of creating an interface for admins to come in and make changes 
- What feature / user story is it for?
- for Admins a new seperate dashboard
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [x ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ x] There are no merge conflicts

### CODE REVIEW

**Comment by ghost** [comment]
Thank you for getting this process started.

File: api/Admin/storiesModel.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const db = require('../../data/db-config');
+// created beginning crud commands for the admin dashboard for Stories
+const getAll = () => {
```
Comment (tinomen) [inline api/Admin/storiesModel.js]: test review

Comment (tinomen) [reply api/Admin/storiesModel.js]: ok

**Review by tinomen — COMMENTED** [review]
This was a test review

**Review by mattcasey879 — APPROVED** [review]
Everything Looks good!

**Review by ashtilawat — APPROVED** [review]
Yes!! Thanks for the fixes @Liliana-Leyva and @mattcasey879!

**Pull request merged**